src/components/Movies.js

```diff
@@ -2,8 +2,55 @@ import React from 'react';
 import Poster from './Poster';
 import './Movies.css';
 
+// class Movies extends React.Component {
+//   constructor({movies}) {
+//     super({movies})
+//     this.movies = {movies};
+//     this.scroller = React.createRef();
```

Comment:
I was initially getting errors while using `ref` in a functional component, so I tried to make `Movies` a class...then I discovered better documentation on refs and the function works fine 🤘

Reply:
for reasons I still don't understand, we have to use `React.createRef()` inside class components and `React.useRef()` inside functional components. Also notable: we can never assign a `ref` TO a functional component, but we can assign a `ref` TO a class component 🤨